src/components/TodoItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User';
+
+const TodoItem = ({ todo }) => {
+  const { id, user, title, completed } = todo;
+
+  return (
+    <tr>
+      <td>{id}</td>
+      <User user={user} />
```

Comment:
```suggestion
      <User userData={user} />
```
Три подряд слова user это слишком много